MOBILE_API.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <?xml-stylesheet type="text/xml" href="protocol2html.xsl"?>
 
-<interface name="SmartDeviceLink RAPI" version="7.0.0" minVersion="1.0" date="2020-11-04">
+<interface name="SmartDeviceLink RAPI" version="7.1.0" minVersion="1.0" date="2020-11-04">
```

Comment:
The version has already been set to 7.1.0 in the develop branch. Please pull in the latest changes from develop.

Reply:
@ShobhitAd Thank you! Done in https://github.com/smartdevicelink/rpc_spec/pull/299/commits/bb4c7fd6dbc79b9c5999c2cad0cd6aef7c93c6be